### PR TITLE
Add the PyGMT talk at AGU24 to the "Overview" section

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

List the PyGMT talk at AGU24 to the "Overview" section

  - [x] Create and add thumbnail image
        - Orientation on existing thumbnails, used a photo @weiji14 made
        - Better link to an image, to avoid including large files into the repo
  - [x] Upload slides to figshare to create a DOI
        - http://dx.doi.org/10.6084/m9.figshare.28049495
        - Recordings are not freely available, do not include an image of the presenter, and are not linked with the slides
  - [x] Upload abstract to ResearchGate
        - https://www.researchgate.net/publication/387127897_PyGMT_-_Accessing_and_Integrating_GMT_with_Python_and_the_Scientific_Python_Ecosystem

**Preview**: https://pygmt-dev--3685.org.readthedocs.build/en/3685/overview.html

**Fixes**: #3683 (partly)

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash command is:

- `/format`: automatically format and lint the code

### CODE REVIEW

**Comment by seisman** [comment]
Two notes:

1. @maxrjones once proposed to move all presentations to the "External Resources" and had some ideas about what to be included in the overview page (https://github.com/GenericMappingTools/pygmt/pull/1764#issuecomment-1048939557). I'm unsure if @maxrjones still remembers the ideas.

2. As previously mentioned in https://github.com/GenericMappingTools/pygmt/pull/2053#issuecomment-1214183407, I feel we should not add large binary files/images into this repository. Even if we decide to add, we should compress the image using tools like https://squoosh.app/

**Comment by yvonnefroehlich** [comment]
> 2. As previously mentioned in https://github.com/GenericMappingTools/pygmt/pull/2053#issuecomment-1214183407, I feel we should not add large binary files/images into this repository. Even if we decide to add, we should compress the image using tools like https://squoosh.app/

Hm. I also was a bit wondering about the the sizes of the image files. But when comparing the image added newly in the PR with the already included images, the file size is in the same range:
overview_images_size

I reduced the size of my image via the tool @seisman mentioned and now it is 140 KB large.

**Comment by yvonnefroehlich** [comment]
> 1. @maxrjones once proposed to move all presentations to the "External Resources" and had some ideas about what to be included in the overview page ([Remove Liam's talk from the overview page #1764 (comment)](https://github.com/GenericMappingTools/pygmt/pull/1764#issuecomment-1048939557)). I'm unsure if @maxrjones still remembers the ideas.

 Yes, adding the conference contributions to the “overview” section is not directly intuitive. But maybe the idea of adding these materials to the overview page was to show how the PyGMT project developed over time (instead of any kind of history section). Some of these contributions are more an overview of the main ideas and aims of the project rather than a more detailed presentation on PyGMT.

**Comment by yvonnefroehlich** [comment]
> These large images were added at the very early stage of the project and I think that's not a good practice.

Then we probably have to discuss / decide if we want to keep / continue the list of conference contributions as is or remove the images from the repo. Or we find a solution where we can just link to an image which serves as thumbnail image.

**Comment by weiji14** [comment]
> Or we find a solution where we can just link to an image which serves as thumbnail image.

If you are planning to upload your slides to Figshare or ResearchGate to create a DOI, you can also upload the thumbnail image there, and we can link to that instead of uploading the thumbnail to the PyGMT repo?

**Comment by yvonnefroehlich** [comment]
> > Or we find a solution where we can just link to an image which serves as thumbnail image.
> 
> If you are planning to upload your slides to Figshare or ResearchGate to create a DOI, you can also upload the thumbnail image there, and we can link to that instead of uploading the thumbnail to the PyGMT repo?

I added the title and summery slides as figures on RG (https://www.researchgate.net/publication/387127897_PyGMT_-_Accessing_and_Integrating_GMT_with_Python_and_the_Scientific_Python_Ecosystem/figures) and linked the title slide for the overview section in the PyGMT docs (commit https://github.com/GenericMappingTools/pygmt/pull/3685/commits/262d6d4986b272f6e4764e0e4a4abf3faac49d90).

**Comment by yvonnefroehlich** [comment]
In commit https://github.com/GenericMappingTools/pygmt/pull/3685/commits/8ffb7887fa5ab929ba29788c05e7e13731c6dc29, I tried to link to the thumbnail images provided by figshare. In principle this works, however the resolution is quite poor. In commit https://github.com/GenericMappingTools/pygmt/pull/3685/commits/12dd4fb251c8dd15c72fc1660875441124714c13, I reduced the width of the images to see if this helps.

**Comment by seisman** [comment]
The easiest solution is to upload the static images into the comment box, and then GitHub will store them for us, just like what we're doing in https://github.com/GenericMappingTools/pygmt/blob/main/doc/external_resources.md,

File: doc/overview.md [hunk 1]
```diff
@@ -32,50 +32,58 @@ our [Discourse forum](https://forum.generic-mapping-tools.org/c/questions/pygmt-
...
+    ![](https://s3-eu-west-1.amazonaws.com/pfigshare-u-previews/51279815/thumb.png){.align-center width="50%"}
+
 -   "Geospatial Analysis & Visualization with PyGMT".
     2022.
     Max Jones, Wei Ji Leong, and Leonardo Uieda.
     Presented at *SciPy 2022*.
     doi:[10.6084/m9.figshare.20483793](https://doi.org/10.6084/m9.figshare.20483793)
 
-    ![](_static/scipy2022-youtube-thumbnail.jpg){.align-center width="80%"}
+    ![](https://s3-eu-west-1.amazonaws.com/pfigshare-u-previews/36664998/thumb.png){.align-center width="50%"}
```
Comment (weiji14) [inline doc/overview.md:49]: For the images files 2022 and older, they have already been commited to the git history, so removing them won't actually reduce the download size when doing a `git clone`. So you can revert the changes here and in the lines below.

File: doc/overview.md [hunk 2]
```diff
@@ -32,50 +32,58 @@ our [Discourse forum](https://forum.generic-mapping-tools.org/c/questions/pygmt-
 
 These are conference presentations about the development of PyGMT (previously "GMT/Python"):
 
+-   "Accessing and Integrating GMT with Python and the Scientific Python Ecosystem".
+    2024.
+	Yvonne Fröhlich, Dongdong Tian, Wei Ji Leong, Max Jones, and Michael Grund.
+	Presented at *AGU 2024*.
+	doi:[10.6084/m9.figshare.28049495](https://doi.org/10.6084/m9.figshare.28049495)
+
+    ![](https://s3-eu-west-1.amazonaws.com/pfigshare-u-previews/51279815/thumb.png){.align-center width="50%"}
```
Comment (weiji14) [inline doc/overview.md:41]: > The easiest solution is to upload the static images into the comment box, and then GitHub will store them for us, just like what we're doing in https://github.com/GenericMappingTools/pygmt/blob/main/doc/external_resources.md,

I use this trick all the time too, but I'm never sure how stable this actually is. What I'd recommend is to upload the full resolution thumbnail image to figshare (see e.g. how I uploaded a PNG thumbnail and PDF version of my [MIGARS poster](https://figshare.com/articles/poster/Applying_geometry-aware_Clifford_Fourier_Neural_Operators_to_weather_forecasting/25599978)) just in case. We can still use the GitHub link, but if that image ever disappears accidentally, we can retrieve it from figshare again.

**Comment by yvonnefroehlich** [comment]
> The easiest solution is to upload the static images into the comment box, and then GitHub will store them for us, just like what we're doing in https://github.com/GenericMappingTools/pygmt/blob/main/doc/external_resources.md,

Actually, I tried this directly at the beginning of this PR in commit https://github.com/GenericMappingTools/pygmt/pull/3685/commits/c7c3228b9eeead073cef9ae9cce7636e2127480c. But for me, the URL related to the image is not in the form `https://user-images.githubusercontent.com/XYZ.png` but in `https://private-user-images.githubusercontent.com/XYZ.png?ABC`. I remember that I faced this issue before, but could not figure out what I have to change on my side.

**Comment by seisman** [comment]
agu2024_talk

Here is my link: `https://github.com/user-attachments/assets/19e1391e-648d-43da-b6f6-ecfb4f3e83e8`

**Review by seisman — APPROVED** [review]

File: doc/overview.md (hunk 2 above)
Comment (yvonnefroehlich) [reply doc/overview.md]: Maybe merge this PR and open a separat issue (and PR) for this?

Comment (seisman) [reply doc/overview.md]: yes, we can discuss it later

Comment (yvonnefroehlich) [reply doc/overview.md]: I opened issue to not forget about this #3708.
